src/shell/device.js

```diff
@@ -122,7 +122,7 @@ export const Battery = GObject.registerClass({
         if (time === 0)
             // TRANSLATORS: When no time estimate for the battery is available
             // EXAMPLE: 42% (Estimating…)
-            return _('%d%% (Estimating…)').format(level);
```

Comment:
These don't need to be changed, since GNOME Shell includes the `String.prototype.format` override.